perf-tool/src/client.cpp

```diff
@@ -162,20 +187,76 @@ void Client::closeClient()
     close(socketFd);
     printf("Closed connection with server.\n");
 }
+void printUsage()
+{
+    printf("Usage: ./a.out [OPTIONS]\n\n"
+            "Options:\n"
+            "--p, --portNo=Port          Specify port number to connect to server\n"
```

Comment:
Does `--p` work? I thought it's either `-p` or `--portNo`

Reply:
Yes `--p` is only working not `-p` for optional arguments. I tested it.